src/cmd/flash.js

```diff
@@ -301,4 +311,43 @@ module.exports = class FlashCommand extends CLICommandBase {
 		}
 		return Array.from(binaries);
 	}
+
+	async _getDeviceOsBinaries({ skipDeviceOSFlash, target, files, firmwareVersion, platformId, applicationOnly }) {
```

Comment:
firmwareversion isn't passed by the caller. What is it for?

Reply:
I wasn't intended hehe. I was testing the function and I forgot to pass again the firmwareVersion. Fixed here! https://github.com/particle-iot/particle-cli/pull/657/commits/624460175e0fd070982c10a681cf8fc7b61a41d5